Week5/week5.py

```diff
@@ -0,0 +1,29 @@
+import csv
+with open ('E:\22bca82\python\student.csv','w') as f:
+    data=csv.writer(f)
+    field=["sid","sname","city","contact"]
+    d=[[1,"heer","kadod",9737762728],[2,"pitu","bardoli",1234567890],[3,"swweta","vyara",9874561230],[4,"suhani","surat",9879461230],[5,"pihu","bardoli",3216549870]]
+    data.writerow(field)
+    data.writerows(d)

```

Comment:
whats difference between writerow and writerows?

Reply:
writerow(): This method writes a single row at a time. Field row can be written using this method and.
 writerows(): This method is used to write multiple rows at a time. This can be used to write rows list.